README.md

```diff
@@ -10,6 +10,22 @@ $ yarn add -D sapui5-runtime
 ```
 This will download and unzip the latest SAPUI5 runtime.
 
+## Installing a specific Version
+In case you need a specific version of SAPUI5 you can specify it in your `package.json` before installing it.
+Add the following line to your `package.json`
+
+```JSON
+"sapui5RuntimeVersion": "X.Y.Z"
```

Comment:
We should use a namespace for our settings to be future proof. I would suggest something like this:
```suggestion
"sapui5-runtime": {
  "version": "X.Y.Z"
}
```

With this change we could add more settings in the future, like specifying a custom download URL etc.